### PR TITLE
Fixed environment variable name on example

### PR DESCRIPTION
**Pull request by matheusneder**
Fixed environment variable name ~(INGEST_ENDPOINT -> AG_INGEST_URL)~ AG_INGEST_URL -> INGEST_ENDPOINT 

**Update**: I did a new commit reverting the INGEST_ENDPOINT -> AG_INGEST_URL rename in fluentd daemonset then renamed AG_INGEST_URL to INGEST_ENDPOIN in configmap (both configmap-activegate.yaml and configmap-saas.yaml) to get the variable name same as confimap key to get code more readable.

**ps**: I believe this change is necessary in order to avoid other people to make the same mistake as I did when following example here (#29).

### CODE REVIEW

**Review by dyladan — APPROVED** [review]
This looks ok to me but I would prefer to wait for @meibensteiner review before merge if possible.

**Comment by arminru** [comment]
Thank you, @matheusneder!

**Pull request merged**
